modules/ROOT/pages/request-reply-scope.adoc

```diff
@@ -41,41 +41,53 @@ Consider a very simple application:
   <logger message="#[payload]" level="INFO"/>
 </flow>
 ----
-<1> After receiving an HTTP request, the VM outbound-endpoint processor sends the message to the /request path.
-<1> The VM inbound-endpoint processor with the /request path configured receives the message triggering the requestFlow processing.
-<1> Only after requestFlow finish processing, Mule automatically sends the message back to VM inbound-endpoint in the reply section of the Request-Reply scope.
+<1> After receiving an HTTP request, the VM outbound-endpoint processor sends the message to the `/request` path.
+<1> The VM inbound-endpoint processor with the `/request` path configured receives the message, triggering the requestFlow processing.
+<1> After `requestFlow` finishes processing, Mule automatically sends the message back to VM inbound-endpoint in the reply section of the Request-Reply scope.
 
-This is possible because Mule implicitly sets a MULE_REPLYTO message property to point to the processor in the reply section of the scope. +
-In this case, the VM inbound-endpoint processor in requestFlow's source receives the message with the MULE_REPLYTO_REQUESTOR property set to "reply", so Mule sends the resulting message (or messages) back to the requester. +
-This is important to note because if the requested flow would have another VM outbound endpoint at the end pointing to a different flow, the VM processor inside the reply scope would still receive the message (or messages) from the triggered flow. +
-You can choose to disable this behavior by setting a MULE_REPLYTO variable:
+To achieve this behavior, Mule implicitly sets a `MULE_REPLYTO` message property to point to the processor in the reply section of the scope. +
+In this case, the VM inbound-endpoint processor in requestFlow's source receives the message with the `MULE_REPLYTO_REQUESTOR` property set to `"reply"`, so Mule sends the resulting message (or messages) back to the requester.
```

Comment:
```suggestion
In this case, the VM inbound-endpoint processor in the requestFlow source receives the message with the `MULE_REPLYTO_REQUESTOR` property set to `"reply"`, so Mule returns the resulting message (or messages) to the requester.
```

Reply:
Are the quotation marks around "reply" part of the syntax? Just checking; you don't have to answer.